src/styles/variables-scss.scss

```diff
@@ -10,7 +10,7 @@
 // $lte-phone-landscape: "only screen and (max-width: 750px) and (orientation: landscape)";
 // $has-hover: "(hover: hover)";
 @import "~ucla-library-design-tokens/scss/variables.scss";
-// @import "~ucla-library-design-tokens/scss/typography.scss";
+@import "~ucla-library-design-tokens/scss/typography.scss";
```

Comment:
we need to keep this commented like before.

Reply:
Also do not commit your package-lock.json, once you have the linux machine, you can try npm install again.

Reply:
But I'm using somethings from typography like @include step-1 and step-2